### ISSUE TITLE
updateRemoteComposerBinary requires sudo

### ISSUE THREAD

**Issue by mchojrin** [EasyCorp/easy-deploy-bundle#117]
Hi:

  I'm trying to deploy my application using this bundle, it works pretty fine generally but now I found I need to do an update to my composer binary.

  I set `updateRemoteComposerBinary` to true but I'm getting an error from trying to place the composer binary inside `/usr/local/bin/`

  It would work if I could just run this command using sudo... Is there any way to set this?

  Thanks!

**Comment by fd6130** [comment]
Maybe you need to set a symlink for your composer.

**Comment by mchojrin** [comment]
Yeah... I don't think that'd work. The problem I have is I'd have to replace the binary stored at `/usr/local/bin/` (which I can do using sudo). Perhaps there could be a new method executeAsSudo or maybe a parameter to use sudo for the command...

**Comment by fd6130** [comment]
Well... i suggest not to update the composer unless you really want to. If the version is working, just kept it as it.

**Comment by mchojrin** [comment]
Ok, thanks.
